products/views.py

```diff
@@ -1,3 +1,111 @@
-from django.shortcuts import render
+from operator import itemgetter
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.db.models import Avg
+from django.views     import View
+
+from django.db.models.aggregates import Avg
+
+from products.models import Category, Product, Image
+from ratings.models  import Rating
+
+class MainView(View):
```

Comment:
우리는 Main 페이지에 해당하는 api 를 생성하는 것이 아닌, 자원단위에서 product 에 해당하는 api 를 생성하고 있습니다.
class 명도 그러한 내용을 담고 있도록 작성해주시면 좋겠습니다.
```suggestion
class ProductView(View):
```

Reply:
반영했습니다.